dts/bindings/adc/nxp,lpc-lpadc.yaml

```diff
@@ -71,8 +71,8 @@ properties:
     required: true
     description: Offset value B to use if CONFIG_LPADC_DO_OFFSET_CALIBRATION is false
 
-  nxp,reference-supply:
```

Comment:
I renamed this property and its type (may also can be considered as deprecated and add a new property), should I add this information to the doc/releases/release-notes-3.7.rst? Seems 3.7 has already been tagged.